### PR TITLE
Add !uwu

### PR DESCRIPTION
**Pull request by Noxium**
I'm sorry

### CODE REVIEW

**Comment by Noxium** [comment]
I'm sorry

**Comment by Noxium** [comment]
`!uwu` will uwu-ize the message from 1 above where the command was run from
`!uwu <messageID>` will usu-ize the message associated with the message ID
`!uwu <messageURL>` will uwu-ize the message associated with the message URL
`!uwu <arbitrary text>` will uwu-ize the text that comes after the command

**Comment by Ramsfield** [comment]
This is disgusting and I love it. Tested on my end, worked as intended. :+1:

File: bot.py [hunk 1]
```diff
@@ -34,6 +35,93 @@
 
 bot = commands.Bot(command_prefix='!')
 
+#variables needed for !uwu
+#I mean what else would it be for?
+suffixes = [
```
Comment (Ramsfield) [inline bot.py:40]: We are currently using a json file as a database. What are your thoughts on dumping suffixes into a "suffix" object of our json library for consistency?

**Review by Ramsfield — CHANGES_REQUESTED** [review]

**Review by Ramsfield — APPROVED** [review]
Yee.

Adds weird complexity for now, but I personally (@CodeMastr3 please let me know if you disagree) would like to see us move towards a more db driven model. 👍

**Comment by CodeMastr3** [comment]
Looking interesting and seems fine

File: bot.py [hunk 2]
```diff
@@ -326,6 +326,10 @@ async def roll(ctx, arg1="1", arg2="100"):
     else:
         await ctx.send(message)
 
+@bot.command(pass_context=True)
```
Comment (Ramsfield) [inline bot.py:329]: I'm gonna vomit, but i approve

**Pull request merged**